docs/application/dotnet/tutorials/overview.md

```diff
@@ -0,0 +1,27 @@
+# Introduction to Tizen .NET Application
+
+Tizen .NET is an exciting new way to develop applications for the Tizen operating system, running on 50 million Samsung devices, including TVs, wearables, mobile phones, and many other IoT devices around the world.
+
+The existing Tizen frameworks are either C-based with no advantages of a managed runtime, or HTML5-based with fewer features and lower performance than the C-based solution. With Tizen .NET, you can use the C# programming language, the Common Language Infrastructure standards, and benefits from a managed runtime for faster application development and code execution, which is efficient and secure.
```

Comment:
Same sentence with dotnet/index.md's. I guess we have same issue with sentence's in the dotnet/index.md.